src/cockpit/superuser.py

```diff
@@ -168,7 +168,7 @@ async def go(self, name: str, responder: ferny.InteractionResponder) -> None:
         except KeyError as exc:
             raise bus.BusError('cockpit.Superuser.Error', f'Unknown superuser bridge type "{name}"') from exc
 
-        self.current = 'init'
+        self.current = f'init-{name}'
```

Comment:
The value "init" is special.  Look in `shell/superuser.jsx`:

```js
    const show = superuser_proxy.Current != "root" && superuser_proxy.Current != "init";
```
```

Reply:
lis | or maybe like, the peer object is already there, no? you could ask it its name

Reply:
Turns out `SuperuserPeer` does not actually know its own name, so I added a new `self.peer_name` state. This is a bit more unwieldy, but avoids changing the `current` D-Bus property. I re-tested this with the devel scenario, and it works fine.